HelloWorld/Main.java

```diff
@@ -1,5 +1,9 @@
-int target = -5;
-int num = 3;
+package HelloWorld;
 
-target = -nua;  
-target += num;
+public class Main
+{
+    public static void main(String[] args)
+    {
+       System.out.println("Hello world");
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this useless assignment to local variable "unread". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.alumni-services-001.com/coding_rules#rule_key=squid%3AS1854)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused "unread" local variable. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.alumni-services-001.com/coding_rules#rule_key=squid%3AS1481)